.github/workflows/develop.yml

```diff
@@ -269,6 +269,13 @@ jobs:
       - name: Install cargo-gc-bin
         shell: bash
         run: cargo install cargo-gc-bin
+      - name: Check aws-lc-sys will not build
+        shell: bash
+        run: |
+             if cargo tree -i aws-lc-sys -e features | grep -q aws-lc-sys; then
```

Comment:
Is cargo tree available here?

Reply:
> Is cargo tree available here?

cargo tree is now a cargo command, so if cargo is installed it should be supported